docs/Actions.md

```diff
@@ -1746,6 +1746,17 @@ It can process unit test results if formatted as junit report as shown in [xctes
 
 ## Misc
 
+### rsync
+
+a wrapper around rsync.
```

Comment:
Instead it would be nice to just describe what it's for (e.g. file transfer) so that someone who's not familiar with `rsync` yet knows what it's about